website/util/rubeus.py

```diff
@@ -97,7 +98,8 @@ def build_addon_root(node_settings, name, permissions=None,
         auth = permissions
         permissions = {
             'view': node_settings.owner.can_view(auth),
-            'edit': node_settings.owner.can_edit(auth) and not node_settings.owner.is_registration
+            'edit': node_settings.owner.can_edit(auth) and not node_settings.owner.is_registration,
+            'diskSavingMode': DISK_SAVING_MODE if node_settings.config.short_name == 'osfstorage' else False
```

Comment:
Instead of adding `diskSavingMode`, it might make sense to add these conditions to `edit`, i.e. `edit: False` if DISK_SAVING_MODE is `True` and the addon is `osfstorage`.